recordset/recordset.js

```diff
@@ -95,11 +97,13 @@
             $rootScope.location = $window.location.href;
         });
 
+        // TODO: We might not need to wrap functions if $on("error", ...) will catch errors thrown from one time function events
         $scope.$on('error', function(event, exception) {
```

Comment:
I noticed that in `ellipses.js` we are `emitting` errors when they occur and then calling the catchAll function. It's becoming more apparent to me that this is most likely unnecessary.